### ISSUE TITLE
Error when loading PyMNE

### ISSUE THREAD

**Issue by likanzhan** [beacon-biosignals/PyMNE.jl#17]
Not error were reported when I added and precompile `PyMNE`.
But when I tried to use it, I got the following error:
```
julia> using PyMNE

signal (11): Segmentation fault: 11
in expression starting at REPL[3]:1
_ZN4llvm17PMTopLevelManager16addImmutablePassEPNS_13ImmutablePassE at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libLLVM-11.dylib (unknown line)
Allocations: 5952326 (Pool: 5949181; Big: 3145); GC: 6

signal (11): Segmentation fault: 11
in expression starting at REPL[3]:1
_PyUnicode_FromId at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libpython3.10.dylib (unknown line)
wait_for_thread_shutdown at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libpython3.10.dylib (unknown line)
Py_FinalizeEx at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libpython3.10.dylib (unknown line)
Py_FinalizeEx at /Users/likanzhan/.julia/packages/PythonCall/dsECZ/src/cpython/pointers.jl:299 [inlined]
#36 at /Users/likanzhan/.julia/packages/PythonCall/dsECZ/src/cpython/context.jl:192
unknown function (ip: 0x14ea6432f)
ijl_apply_generic at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
_atexit at ./initdefs.jl:372
jfptr__atexit_29809 at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
ijl_atexit_hook at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
ijl_exit at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
jl_exit_thread0_cb at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
Allocations: 5952326 (Pool: 5949181; Big: 3145); GC: 6

```

And here is the version information of my Julia:

```
Julia Version 1.8.5
Commit 17cfb8e65ea (2023-01-08 06:45 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin21.5.0)
  CPU: 10 × Apple M1 Max
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, apple-m1)
  Threads: 4 on 8 virtual cores
Environment:
  JULIA_NUM_THREADS = 4
  JULIA_PKG_USE_CLI_GIT = true
```

**Comment by palday** [comment]
I can reproduce this in a clean environment but I don't yet know what causes it. Thanks for reporting!

**Comment by palday** [comment]
Okay, this looks like to be an M1 issue -- no trouble on x86-64. I have my suspicions about what causes this, but I suspect it's a Python 3.10 / PythonCall issue.

**Comment by palday** [comment]
@likanzhan can you test if you're able to use the fix in #18?

```julia
pkg> install PyMNE#pa/depup
julia> using PyMNE
```

**Comment by marcpabst** [comment]
This indeed fixes the issue for me (on both M1 and M2 Pro). I think this should be pushed to master.
